src/main/java/it/eng/idsa/businesslogic/web/rest/PasswordController.java

```diff
@@ -0,0 +1,47 @@
+package it.eng.idsa.businesslogic.web.rest;
+
+import io.swagger.v3.oas.annotations.Hidden;
+import it.eng.idsa.businesslogic.service.impl.PasswordValidatorService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * @author Antonio Scatoloni on 22/04/2020
+ **/
+
+@RestController
+@EnableAutoConfiguration
+@RequestMapping({"/notification"})
```

Comment:
Can we move this to /api/password instead notification?
Also update documentation and https config (SecurityConfig.java?)